merkletree/pad_test.go

```diff
@@ -264,10 +264,23 @@ func TestTB(t *testing.T) {
 	// create next epoch and see if the TB is inserted as promised:
 	pad.Update(nil)
 
-	ap, err := pad.Lookup(key1)
+	ap, err := pad.Lookup(key)
 	if !bytes.Equal(ap.LookupIndex, tb.Index) || !bytes.Equal(ap.Leaf.Value(), tb.Value) {
 		t.Error("Value wasn't inserted as promised")
 	}
+	// step 1. verify VRF index
+	if !bytes.Equal(vrfPrivKey1.Compute([]byte(key)), ap.LookupIndex) {
```

Comment:
I think we should verify the VRF index in return `TB` first (after verifying the signature), then we could remove this verification since we already compare both indices above.